cmake/PackageConfig.cmake.in

```diff
@@ -9,6 +9,11 @@
 # Include targets file.  This will create IMPORTED target @PROJECT_NAME@
 include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
 
+include(CMakeFindDependencyMacro)
+
```

Comment:
what about Jasper dependency?

Reply:
I think there is no need to add Japser as package dependency because dependency on Japser is specified using ``${JASPER_LIBRARIES}`` variable:
```
    target_link_libraries(${lib_name} PUBLIC bacio::bacio_4 PNG::PNG ${JASPER_LIBRARIES})
```
which results in full path being used in package file (not target as bacio and PNG). For example on Orion:
```
set_target_properties(g2::g2_d PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "/work/noaa/fv3-cam/djovic/sufs/simple-ufs/libs/nceplibs/local/g2_3.2.0/include_d"
  INTERFACE_LINK_LIBRARIES "bacio::bacio_4;PNG::PNG;/apps/contrib/NCEPLIBS/orion/lib/libjasper_v1.900.2/libjasper.a;/usr/lib64/libjpeg.so"
)
```
which means there's no need for user's cmake to "find" Japser.